tests/test-data/projects/docker-compose/basic/docker-compose.yml

```diff
@@ -12,3 +12,5 @@ services:
     build:
       context: ./service2
       dockerfile: Dockerfile-alt
+      args:
+        - 'COMPOSE_ARG=an argument defined in the docker-compose.yml file'
```

Comment:
This added variable in this test `docker-compose.yml` file allows us to test that the `--buildArg` command-line argument takes precedence.